packages/components/src/Notification/ToastNotification/_docs/ToastNotification.stories.tsx

```diff
@@ -0,0 +1,173 @@
+        uploaded. <a href="/">Manage users is now available</a>
+      </div>
+    ),
+  },
+} satisfies Meta<typeof ToastNotification>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
+export const CreateNotification: Story = {
+  render: () => (
+    <Button
+      label="Create notification"
+      onClick={() =>
+        addToastNotification({
+          title: "Informative",
+          type: "informative",
+          message: "New notification!",
+        })
+      }
+    />
+  ),
+}
+
+export const UpdateNotification: Story = {
+  render: () => (
+    <>
+      <Button
+        label="Create notification"
+        classNameOverride="!mr-12"
+        onClick={() =>
+          addToastNotification({
+            id: "id--update-example",
+            title: "Cautionary",
+            type: "cautionary",
+            message: "This content will be updated",
+          })
+        }
+      />
+      <Button
+        label="Update notification"
+        onClick={() =>
+          addToastNotification({
+            id: "id--update-example",
+            title: "Success",
+            type: "positive",
+            message: "The content was successfully updated",
+          })
+        }
+      />
+    </>
+  ),
+}
+
+export const RemoveNotification: Story = {
+  render: () => (
+    <>
+      <Button
+        label="Create notification"
+        classNameOverride="!mr-12"
+        onClick={() =>
+          addToastNotification({
+            id: "id--remove-example",
+            title: "Remove",
+            type: "negative",
+            message: "This notification will be removed",
+          })
+        }
+      />
+      <Button
+        label="Remove notification"
+        onClick={() => removeToastNotification("id--remove-example")}
+      />
+    </>
+  ),
+}
+
+export const ClearNotifications: Story = {
+  render: () => (
+    <>
+      <Button
+        label="Create notifications"
+        classNameOverride="!mr-12"
+        onClick={() => {
+          addToastNotification({
+            id: "id--clear-example-1",
+            title: "First",
+            type: "positive",
+            message: "This notification will be removed",
+          })
+          addToastNotification({
+            id: "id--clear-example-2",
+            title: "Second",
+            type: "cautionary",
+            message: "This notification will also be removed",
+          })
+          addToastNotification({
+            id: "id--clear-example-3",
+            title: "Third",
+            type: "negative",
+            message: "This notification will also also be removed",
+          })
+        }}
+      />
+      <Button
+        label="Clear notifications"
+        onClick={() => clearToastNotifications()}
+      />
+    </>
+  ),
+}
+
+export const MultipleNotifications: Story = {
```

Comment:
This doesn't appear used (we don't expose individual stories on prod, only the Docs page). Was this meant to be the stickersheet?

Reply:
Actually I should remove this, it's an old story but I don't think there's any valid use case for it that hasn't been tested in other stories

Reply:
Tis' gone with the wind 🎐